aicssegmentation/structure_wrapper/WorkflowStep.py

```diff
@@ -43,11 +43,10 @@ def execute(self, image: List) -> np.ndarray:
         if self.__parameters:
             self.result: np.ndarray = self.__function(*image, **self.__parameters)
         else:
-            # for "Merge Segmentation' we need a list of images, while other functions take in with multiple params
-            if self.name == "Merge Segmentation":
-                # feed in as list
-                self.result: np.ndarray = self.__function(image)
-            else:
-                # unpack list and feed in images
+            try:
+                # Most functions require unpacking the images into individual params
                 self.result: np.ndarray = self.__function(*image)
+            except:
```

Comment:
If we're sticking with this version - we should only catch the proper exception instead of a blanket `except`.
So something like this:
`except Typerror:`